index.yaml

```diff
@@ -14835,6 +14835,20 @@ entries:
     - assets/rancher-logging/rancher-logging-3.6.000.tgz
     version: 3.6.000
   rancher-logging-crd:
+  - annotations:
```

Comment:
This PR added a new entry and it shouldn't. I'll keep this one and delete the other one in release-v2.8